src/D2L.Bmx/ConsolePrompter.cs

```diff
@@ -32,6 +32,16 @@ internal static class ConsolePrompter {
 		return user;
 	}
 
+	public static int PromptDefaultDuration() {
+		Console.Write( "Default duration of session in minutes (recommended: 60): " );
```

Comment:
instead of "recommended: 60", I wonder if we should just say "default 60", and allow the user to not enter anything (and we don't write anything to the config file in that case)